zboss/CMakeLists.txt

```diff
@@ -38,14 +43,42 @@ if ((NOT DEFINED ZEPHYR_PLATFORM_NCS_MODULE_DIR) OR
                     "(${ZBOSS_LIB_PATH} doesn't exist.)")
   endif()
 
+  target_include_directories(zboss INTERFACE
+    include
+    include/zcl
+    include/ha
+    include/se
+    include/osif
+  )
+
+  # Define the ZBOSS imported library, that contains only static libraries
   if (CONFIG_ZIGBEE_ROLE_COORDINATOR OR CONFIG_ZIGBEE_ROLE_ROUTER)
-    zephyr_link_libraries(${ZBOSS_LIB_PATH}/libzboss.a)
-    toolchain_cc_imacros(${CMAKE_CURRENT_SOURCE_DIR}/include/osif/libzboss_config.h)
+    zephyr_library_import(zboss-libraries ${ZBOSS_LIB_PATH}/libzboss.a)
+    set(LIBZBOSS_CONFIG_FILE "libzboss_config.h")
```

Comment:
did you mean:
```suggestion
target_compile_definitions(zboss INTERFACE LIBZBOSS_CONFIG_FILE=libzboss_config.h)
```

Reply:
I've included apostrophes too, so you can place this header anywhere you want ;)